git-helpers/src/remote_helper.rs

```diff
@@ -88,7 +88,12 @@ pub fn run() -> anyhow::Result<()> {
 fn get_signer(git_dir: &Path, keys_dir: &Path, url: &LocalUrl) -> anyhow::Result<BoxedSigner> {
     let pass = credential::Git::new(git_dir).get(url)?;
     let file = keys_dir.join(SECRET_KEY_FILE);
-    let keystore = FileStorage::<_, PublicKey, _, _>::new(&file, Pwhash::new(pass));
+    let kdf_params = if cfg!(test) {
+        *radicle_keystore::crypto::KDF_PARAMS_TEST
+    } else {
+        *radicle_keystore::crypto::KDF_PARAMS_PROD
+    };
```

Comment:
:man_shrugging:

Reply:
Can we just use prod here?

Reply:
:man_shrugging: :man_shrugging: :man_shrugging: :man_shrugging: :man_shrugging: :man_shrugging: :man_shrugging: :man_shrugging:

Reply:
Will it even be conditionally compiled? AFAIU the remote helper is invoked as a binary and `cfg(test)` depends on `cargo test`.